.circleci/config.yml

```diff
@@ -216,8 +217,9 @@ jobs:
             [[ "$CIRCLE_BRANCH" == "main" ]] && ECR_TAG="main-$CIRCLE_SHA1" || ECR_TAG="branch-$CIRCLE_SHA1"
             echo "export ECR_TAG=$ECR_TAG" >> "$BASH_ENV"
             source "$BASH_ENV"
-      - aws-ecr/build-image:
-          push-image: true
+      - aws-ecr/build_image:
+          push_image: true
+          account_id: $AWS_ECR_REGISTRY_ID
```

Comment:
Is this related to the version change of the `aws-ecr` orb?

Reply:
Yes this field is now required for build_image
https://circleci.com/developer/orbs/orb/circleci/aws-ecr#commands-build_image

Reply:
and the env_var was already in use in circleci

Reply:
Thanks!